In [ ]:
import math
import scipy.stats as stats

In [ ]:
samplemean_cost=3050 #sample mean weekly cost
mean_units=600 #theoretical mean units produced
std_units=25 # std deviation of units produced
sample_size=25 #25 restaurants
alpha=0.05
theoretical_meancost= 1000 + 5 * mean_units #theoretical cost
std_cost=5*std_units # W= 1000 + 5X

t=(samplemean_cost-theoretical_meancost)/(std_cost/math.sqrt(sample_size))
print("t stat:",t)

critical_value=stats.norm.ppf(1-alpha)
print("crticial value", critical_value)
dec="reject the null hypothesis" if t>critical_value else "accept the null hypothesis"
print("decision:",dec)
conclusion=("There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests."
if dec=="reject the null hypothesis"
else "There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")
print("conclusion:", conclusion)


t stat: -38.0
crticial value 1.6448536269514722
decision: accept the null hypothesis
conclusion: There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.


Null Hypothesis (H₀): The observed weekly operating costs are not higher than the theoretical model.

Alternative Hypothesis (H₁): The observed weekly operating costs are higher than the theoretical model.

Conclusion:

Test Statistic (t): The calculated test statistic is -38.0.

Critical Value (Z): The critical value for a one-tailed test with alpha = 0.05
is 1.645.

Decision: Since the test statistic (-38.0) is not greater than the critical value (1.645), we fail to reject the null hypothesis.

Conclusion: There is not enough evidence to support the restaurant owners' claim that the weekly operating costs are higher than predicted by the cost model.